am-lib/src/test/java/uk/gov/hmcts/reform/amlib/AccessManagementServiceBenchmarkTest.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.amlib;
+
+import org.junit.jupiter.api.Test;
+import org.openjdk.jmh.profile.StackProfiler;
+import org.openjdk.jmh.results.RunResult;
+import org.openjdk.jmh.results.format.ResultFormatType;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+import java.util.Collection;
+
+import static org.junit.Assert.assertTrue;
+
+@SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+class AccessManagementServiceBenchmarkTest {
+
+    private static final double REFERENCE_SCORE = 50.000;
+
+    @Test
```

Comment:
Running this in nightly pipeline might be a challenge as performance test step is hard wired to gatling. We would need to try work around it.